src/renderer/scss/_gui.scss

```diff
@@ -121,6 +121,10 @@ input::placeholder {
   opacity: 0.5;
 }
 
+label {
+  cursor: pointer;
```

Comment:
I'm guessing this ought to be `label[for]`, if not something even stronger or more restrictive.